Source/core/PriviligedRequest.h

```diff
@@ -0,0 +1,360 @@
+/*
+ * If not stated otherwise in this file or this component's LICENSE file the
+ * following copyright and licenses apply:
+ *
+ * Copyright 2022 Metrological
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+// #define MODULE_NAME PriviligedRequest
+#include <core/core.h>
```

Comment:
This file is part of core, so the include should be "part of" the core, and not to be treated as an external include file. Please use: #include "core.h" and start by Including "module.h (see other header files)

Reply:
Fixed!